client/components/BuilderPedals/BuilderPedals.js

```diff
@@ -1,23 +1,51 @@
-import React, { Component } from 'react'
-import { Image } from 'semantic-ui-react'
+import React from 'react'
+import { Image, Icon } from 'semantic-ui-react'
 import Draggable from 'react-draggable'
 
-const BuilderPedals = ({ pedals }) => {
+const BuilderPedals = props => {
   return (
-    <div style={{ position: 'absolute', top: '20%', left: '29%' }}>
-      {pedals.map((elem, index) => (
+    <div
+      style={{
```

Comment:
You should extract non-dynamic styles into a variable outside of your component(s).  Keep your render methods and stateless function components as short as you can.

```js
const styles = {
  pedals: {
    display: 'flex',
    flexDirection: 'row',
    // ...
  },
  icon: {
    fontSize: '40px',
    // ...
  }
}
```

```jsx
<div style={styles.pedals}/>

<Icon style={styles.icon}/>
```